### ISSUE TITLE
Sanitize requests with no content type

### ISSUE THREAD

**Issue by jakeonfire** [whitequark/rack-utf8_sanitizer#50]
We had lots of web requests circumventing our installation of `rack-utf8_sanitizer`, and it turned out they were for additional resources (`css`, `js`, etc) and had _no_ content type. Luckily, the following configuration worked for us:
```ruby
config.middleware.insert 0, Rack::UTF8Sanitizer, additional_content_types: [nil]
```
Perhaps this is worth mentioning in the README. Has anyone else run into this issue?

**Comment by ndbroadbent** [comment]
@jakeonfire Thanks for the tip! I've just installed `rack-utf8_sanitizer`, and I'm checking out the issues to see if there are any problems to be aware of.

I was wondering what kind of UTF-8 errors you were seeing for your assets? Is this for the paths or query params?

**Comment by jakeonfire** [comment]
@ndbroadbent yeah, we were getting "ArgumentError: invalid byte sequence in UTF-8" errors from query string keys, which is likely quite borderline. but try
```ruby
get '/?%ff=1', headers: { 'Content-Type' => nil }
```
